Argument based debug

KenwoodFox opened the pull request:
A proposed approach to making testing and debugging easier by using arguments to set the IP, the code should have a default IP but use any IP you pass as an argument, does this work in a launch file?

KenwoodFox committed:
Add code to handle arguments

KenwoodFox committed:
Add import

mikeferguson commented on src/tidalforce_odom/src/odom.py:
```diff
@@ -127,8 +128,13 @@ def update(self, left, right):
+    #ip = rospy.get_param("~ip", "localhost") # For testing
     #ip = rospy.get_param("~ip", "10.17.21.55") # First mDNS almost never works on the feild
-    ip = rospy.get_param("~ip", "localhost") # For testing
+    if (len(sys.argv) > 1): # If there is more than one argument
```
you don't actually need the outer parenthesis.

KenwoodFox replied:
So I should just use `if len(sys.argv) > 1:` right?

mikeferguson commented on src/tidalforce_odom/src/odom.py:
```diff
@@ -29,6 +29,7 @@
 """
 
 import rospy
+import sys
```
for more complex command line arguments - and automatic "help" command, check out argparse.

mikeferguson reviewed (approved):
LGTM

KenwoodFox committed:
Removed extra parenthsies

Relates to comment on pull request #9
Was unable to identify argparse use, will implement at a later date
